info.febvre.Komikku.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "info.febvre.Komikku",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "komikku",
+    "finish-args" : [
+        "--socket=session-bus",
```

Comment:
Run the app with `--log-session-bus` and set the right DBus names you need instead of session-bus here.

Reply:
Initially, it was to avoid GNOME Builder WARNING/DEBUGs:

```
(komikku:3): dbind-WARNING **: 22:52:36.449: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
(komikku:3): GVFS-RemoteVolumeMonitor-DEBUG: 22:52:36.541: Error: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(komikku:3): GVFS-DEBUG: 22:52:36.541: cannot open directory /usr/share/gvfs/remote-volume-monitors: Error opening directory “/usr/share/gvfs/remote-volume-monitors”: No such file or directory
(komikku:3): GVFS-DEBUG: 22:52:36.546: org.gtk.vfs.MountTracker.listMountableInfo call failed: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown (g-dbus-error-quark, 2)

(komikku:3): GLib-GIO-DEBUG: 22:52:36.546: _g_io_module_get_default: Found default implementation local (GLocalVfs) for ‘gio-vfs’
(komikku:3): GLib-GIO-DEBUG: 22:52:36.546: _g_io_module_get_default: Found default implementation keyfile (GKeyfileSettingsBackend) for ‘gsettings-backend’
```

Output of `-log-session-bus` isn't super-easy to understand. Idem if I run `dbus-monitor --session` command.

Perhaps
```
"--talk-name=org.gtk.vfs.*"
```

but  I'm not sure. App seems to work without that.